### PR TITLE
Club approval response templates

### PR DESCRIPTION
**Pull request by rm03**
This PR introduces a new feature that enables site administrators to efficiently manage templated responses for the club approval/rejection process. Admins can now create, update, and select predefined templates when approving or rejecting clubs (works for individual or bulk approvals/rejections). This feature replaces the manual task of copying and pasting responses, reducing overhead and ensuring consistent messaging

Template management tab on admin dashboard:
image

Template dropdown on club page:
image

Template dropdown on queue page for bulk approvals/rejections:
image

### CODE REVIEW

File: backend/tests/clubs/test_views.py [hunk 1]
```diff
@@ -2856,3 +2857,69 @@ def test_event_add_meeting(self):
         self.event1.refresh_from_db()
         self.assertIn("url", resp.data, resp.content)
         self.assertTrue(self.event1.url, resp.content)
+
+    def test_club_approval_response_templates(self):
```
Comment (aviupadhyayula) [inline backend/tests/clubs/test_views.py:2861]: Great tests 🙂

**Review by aviupadhyayula — APPROVED** [review]
This is sick, great work Rohan. Backend code LGTM. About the frontend: do we plan to support someone applying multiple templates to the same approval decision? I can envision a club getting rejected for multiple reasons (e.g. a club's name has Penn in it, and it doesn't have enough members).

**Comment by rm03** [comment]
> This is sick, great work Rohan. Backend code LGTM. About the frontend: do we plan to support someone applying multiple templates to the same approval decision? I can envision a club getting rejected for multiple reasons (e.g. a club's name has Penn in it, and it doesn't have enough members).

Just added this, if one of you can test locally and let me know if it works that would be great 🙂

**Review by julianweng — APPROVED** [review]
Tested locally, some code duplication is acceptable given the scale so LGTM!

**Pull request merged**